packages/strapi-plugin-content-manager/admin/src/utils/removeFieldsFromClonedData.js

```diff
@@ -0,0 +1,75 @@
+import { get } from 'lodash';
+import { getType, getOtherInfos } from './getAttributeInfos';
+
+const defaultFields = [
+  'created_at',
+  'createdAt',
+  'created_by',
+  'createdBy',
+  'updated_at',
+  'updatedAt',
+  'updated_by',
+  'updatedBy',
+  'published_at',
+  'id',
+  '_id',
+];
```

Comment:
What about custom timestamps? 
Shouldn't we get the content type timestamp values?